### ISSUE TITLE
Can't move windows inside application

### ISSUE THREAD

**Issue by kylefelipe** [pgmodeler/pgmodeler#1100]
Hi guys!
There is a way to set floating windows??? some times i need see what is behind and i cant change its place, or monitor (i work with 2 here).
S.O: Debian Stretch
PgModeler Version: 0.9.1

**Comment by colincci** [comment]
This is a real problem for me.  Not being able to move the in-application windows to see what is behind them is seriously hampering the overall usability of this otherwise EXCELLENT tool.
I can't believe there are not more complaints about this.
Is this something unique to certain installs?  Is there a way to work around this behaviour?

**Comment by rkhaotix** [comment]
Hi! I don't get it.

You can move internal will over the screen. Well, at least in my development environment: KDE 5 on openSus 15. Also, on Windows 10 and macOs everything is okay.

**Comment by kylefelipe** [comment]
@rkhaotix here, i use 2 screens to work...
But i cant move an window to the second screen or see what is behind it, and when i do it, parent window changes with...

**Comment by colincci** [comment]
Yes, I have the same problem.  If I grab the in-application window to move it, the parent window behind collapses to the same size and moves with the in-application window.  Very troublesome.  I have confirmed this behavior on Gnome 3 on Fedora 27, RHEL 7.5 and Debian Stable.  I have not tried with KDE.  Perhaps this is a Gnome 3 issue?

Thanks

Colin

**Comment by colincci** [comment]
Hi Folks

So, this is a Gnome 3 problem.  Apparently, 'NOT A BUG'.  Sigh!!  They have set the default for modal windows to be attached.  This completely breaks the usability of PgModeler under gnome3.  I am surprised it hasn't shown up in more apps that I use.  I tested under Mate and Pgmodeler  works fine.  No problems.  

The solution under Gnome 3, is to install the Gnome Tweak Tool (really a necessity for gnome 3 anyway) and then go to Windows and set the 'Attached Modal Dialogs' to OFF.  Then the in-applications windows in Pgmodeler behave as they should.

Thanks everyone.  It's unlikely I would have solved this problem if it wasn't for this github issue.  I would have, unfortunately, assumed it was a pgmodeler problem.

Thanks for the GREAT work on an excellent piece of software.

Colin

**Comment by kylefelipe** [comment]
Works like a charm...

**Comment by Maxzor** [comment]
Thanks a lot for that one!
